tools/reg_parse.c

```diff
@@ -117,10 +119,12 @@ pre_parse( char * dir, FILE * infile, FILE * outfile )
     for ( p = inln ; ( *p == ' ' || *p == '	' ) && *p != '\0' ; p++ ) ;
     if ( !strncmp( p , "include", 7 ) &&  ! ( ifdef_stack_ptr >= 0 && ! ifdef_stack[ifdef_stack_ptr] ) ) {
       FILE *include_fp ;
-      char include_file_name_local_registry[128] ;
-      char include_file_name[128] ;
+      char include_file_name_local_registry[MAX_PATH] ;
+      char include_file_name[MAX_PATH] ;
       p += 7 ; for ( ; ( *p == ' ' || *p == '	' ) && *p != '\0' ; p++ ) ;
-      if ( strlen( p ) > 127 ) { fprintf(stderr,"Registry warning: invalid include file name: %s\n", p ) ; }
+      fprintf( stderr, "Len : %d\n", strlen( p ) );
+      if ( strlen( p ) > MAX_PATH - 1 ) { fprintf(stderr,"Registry error: include file name too long: %s\n", p ) ; }
```

Comment:
For consistency with other messages throughout the registry tool, should we retain the "Registry warning:" text for now rather than "Registry error:"? A separate PR could be used to consistently change "warning" messages to "error" messages throughout the `reg_parse.c` code, and at that time, we could also decide whether to use the word "Error" (with a capital-e), as it seems like a lot of guidance to WRF users is to search for "Error" in the build log.

Reply:
I understand the sentiment, though I think there is something to be said for piecemeal improvements that don't significantly impact functionality while incomplete. I think this could fall under a reasonable clean-as-you-go improvement to better the message output to reflect the severity.

There isn't exactly a good precedent set by registry message prefixes both in style and actual severity. Given the lack of consistency in registry messages, I think it would be better to update messages to say what they actually mean whilst editing the relevant code rather than keeping consistent with the most used prefix to be fixed later.

The fix to update message prefixes should still happen, but I don't think that fix should preclude us from making incremental progress.